src/Card/CardStatus.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from '..';
+
+const CardStatus = React.forwardRef(({
+  className,
+  children,
+  variant,
+  icon,
+  title,
+}, ref) => (
+  <div
+    className={classNames('pgn__card-status',
+      `alert-${variant}`, 'alert', 'alert-content',
+      className)}
+    ref={ref}
+  >
+    {icon && <Icon src={icon} className="alert-icon" />}
+    <div className="alert-message-content">
+      {title && <div className="alert-heading h4">{title}</div>}
+      {children && <p>{children}</p>}
```

Comment:
I think we can make `children` a required prop and always render it here.

Reply:
Yes, agree, added